packages/apalis-sql/src/postgres.rs

```diff
@@ -681,6 +682,23 @@ mod tests {
         cleanup(storage, &worker_id).await;
     }
 
+    #[tokio::test]
+    async fn test_schedule_job() {
+        let mut storage = setup().await;
+        let time = Utc::now() + Duration::from_secs(1);
```

Comment:
Is it possible to add a larger number of seconds, ensure its not started, sleep then check again?

Reply:
What about now ?